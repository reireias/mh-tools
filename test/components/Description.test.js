import { mount, createLocalVue } from '@vue/test-utils'
import Vue from 'vue'
import Vuetify from 'vuetify'
import target from '@/components/Description'

Vue.use(Vuetify)
const localVue = createLocalVue()
localVue.use(Vuetify)

describe('Description', () => {
  it('should be vue instance', () => {
    const vuetify = new Vuetify()
    const wrapper = mount(target, {
      localVue,
      vuetify,
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
