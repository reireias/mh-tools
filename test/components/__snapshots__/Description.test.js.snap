// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description should be vue instance 1`] = `
"<div class=\\"v-item-group theme--light v-expansion-panels v-expansion-panels--popout\\">
  <div aria-expanded=\\"false\\" class=\\"v-expansion-panel\\"><button type=\\"button\\" class=\\"v-expansion-panel-header\\">基礎攻撃力の計算方法<div class=\\"v-expansion-panel-header__icon\\"><i aria-hidden=\\"true\\" class=\\"v-icon notranslate mdi mdi-chevron-down theme--light\\"></i></div></button>
    <div class=\\"v-expansion-panel-content\\" style=\\"display: none;\\"></div>
  </div>
</div>"
`;
